go/cmd/dolt/commands/show.go

```diff
@@ -112,6 +112,10 @@ func (cmd ShowCmd) ArgParser() *argparser.ArgParser {
 	return ap
 }
 
+func (cmd ShowCmd) RequiresRepo() bool {
+	return false
```

Comment:
This (and the change to diff) feel unrelated to enabling remote connections and probably belong in separate PR.

Reply:
I see now that the bats test uses them. These should _probably_ be flipped on in a separate RP and then merged into this one... but I don't think it's a big deal either way.

Reply:
They should have been enabled in the original migration. Making another PR seems like overkill.